src/vstt/stats.py

```diff
@@ -542,26 +630,57 @@ def preprocess_mouse_positions(mouse_positions: np.ndarray) -> np.ndarray:
     return mouse_positions
 
 
-def _peak_velocity(mouse_times: np.ndarray, mouse_positions: np.ndarray) -> float:
+def _peak_velocity(
```

Comment:
Now that peak_velocity has both a value and an index, either
- `append_stats_data_to_excel` needs updating to split this into two columns, as is already done for e.g. target_pos
- or it could be two different columns, e.g. the existing `peak_velocity` and a new `peak_velocity_index`